ethereum/blockwatch/block_watcher_test.go

```diff
@@ -1,3 +1,5 @@
+// +build !browser
```

Comment:
This build tag means the test won't run in browser environments. It wouldn't work because we try to load a JSON file.